src/resolver/dock-resolver.js

```diff
@@ -0,0 +1,19 @@
+import DockDIDResolver from './did/dock-did-resolver';
+import DockStatusListResolver from './status-list2021/dock-status-list2021-resolver';
+import { MultiResolver } from './generic';
+
+/**
+ * Resolves dock-hosted entities such us `did:dock:*` and `status-list2021:dock:*`.
+ */
+export default class DockResolver extends MultiResolver {
+  static PREFIX = [DockDIDResolver.PREFIX, DockStatusListResolver.PREFIX];
```

Comment:
would it make sense to have a resolver for the previous revocation strategy here too or do you just not want to bother since its legacy? (fine either way)